File: cpp/07/main.cpp
#include <iostream>

using namespace std;

// 输入数据有多组, 每行表示一组输入数据。
// 每行不定有n个整数，空格隔开。(1 <= n <= 100)。

// 每组数据输出求和的结果

int main() {
    int n, sum;
    while (cin >> n) {
        sum += n;
        if (cin.get() == '\n') {
            cout << sum << endl;
            sum = 0;
        }
    }
    return 0;
}

File: go/07/main.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func main() {
	reader := bufio.NewReader(os.Stdin)
	for {
		data, err := reader.ReadString('\n')
		if err != nil {
			break
		}
		data = data[:len(data)-1]
		vstr := strings.Fields(data)
		sum := 0
		for _, v := range vstr {
			vi, _ := strconv.Atoi(v)
			sum += vi
		}
		fmt.Println(sum)
	}
}
